tests/dummy/app/templates/nodes/index.hbs

```diff
@@ -18,7 +18,7 @@
     {{input type="text" value=title placeholder="Node's title" required=true}}
     <label>Description (optional):</label>
     {{input type="text" value=description placeholder="Node's description"}}
-    <p><button {{action "createNode" title description}}>Create Node</button></p>
+    <p><button onclick={{action 'createNode' title description}}>Create Node</button></p>
```

Comment:
It's really not necessary. I kind of prefer the explicit `onclick` just because <button> is native html, but this could be removed.